Workbooks/Windows Virtual Desktop/shared/Host Pool Overview/Host Pool Overview.workbook

```diff
@@ -250,7 +265,7 @@
     "query": "{\"version\":\"Merge/1.0\",\"merges\":[{\"id\":\"e0297506-948a-4021-b236-175107e751db\",\"mergeType\":\"leftouter\",\"leftTable\":\"HostPools\",\"rightTable\":\"hostpoolspecifics\",\"leftColumn\":\"HostPool\",\"rightColumn\":\"HostPool\"},{\"id\":\"e0297506-948a-4021-b236-175107e751dc\",\"mergeType\":\"leftouter\",\"leftTable\":\"HostPools\",\"rightTable\":\"HostUsers\",\"leftColumn\":\"Name\",\"rightColumn\":\"Computername\"}],\"projectRename\":[{\"originalName\":\"[HostPools].ResoourceId\",\"mergedName\":\"Host Name\",\"fromId\":\"e0297506-948a-4021-b236-175107e751db\"},{\"originalName\":\"[HostPools].Status\",\"mergedName\":\"Status\",\"fromId\":\"e0297506-948a-4021-b236-175107e751db\"},{\"originalName\":\"[HostPools].LastHeartBeat\",\"mergedName\":\"LastUpdateStatus\",\"fromId\":\"unknown\"},{\"originalName\":\"[HostPools].AllowNewSessions\",\"mergedName\":\"AllowNewSessions\",\"fromId\":\"e0297506-948a-4021-b236-175107e751db\"},{\"originalName\":\"[HostPools].StackVersion\",\"mergedName\":\"StackVersion\",\"fromId\":\"e0297506-948a-4021-b236-175107e751db\"},{\"originalName\":\"[HostPools].HostPool\",\"mergedName\":\"HostPool\",\"fromId\":\"e0297506-948a-4021-b236-175107e751db\"},{\"originalName\":\"[HostPools].Sessions\",\"mergedName\":\"Sessions\",\"fromId\":\"e0297506-948a-4021-b236-175107e751db\"},{\"originalName\":\"[hostpoolspecifics].MaxSessionLimit\",\"mergedName\":\"MaxSessionLimit\",\"fromId\":\"e0297506-948a-4021-b236-175107e751db\"},{\"originalName\":\"[Added column]\",\"mergedName\":\"AvailableSessions\",\"fromId\":null,\"isNewItem\":true,\"newItemData\":[{\"criteriaContext\":{\"leftOperand\":\"Sessions\",\"operator\":\"<=\",\"rightValType\":\"column\",\"rightVal\":\"MaxSessionLimit\",\"resultValType\":\"expression\",\"resultVal\":\"[\\\"MaxSessionLimit\\\"] -[\\\"Sessions\\\"]\"}},{\"criteriaContext\":{\"operator\":\"Default\",\"rightValType\":\"column\",\"resultValType\":\"static\",\"resultVal\":\"0\"}}]},{\"originalName\":\"[HostUsers].Users\",\"mergedName\":\"Users\",\"fromId\":\"e0297506-948a-4021-b236-175107e751dc\"},{\"originalName\":\"[HostPools].URI\",\"mergedName\":\"URI\",\"fromId\":\"unknown\"},{\"originalName\":\"[HostPools].OSversion\",\"mergedName\":\"OSversion\",\"fromId\":\"unknown\"},{\"originalName\":\"[HostPools].AssignedUser\",\"mergedName\":\"AssignedUser\",\"fromId\":\"unknown\"},{\"originalName\":\"[HostPools].Name\"},{\"originalName\":\"[hostpoolspecifics].resourceGroup\"},{\"originalName\":\"[hostpoolspecifics].HostPool\"},{\"originalName\":\"[hostpoolspecifics].location\"},{\"originalName\":\"[hostpoolspecifics].LoadBalanceType\"},{\"originalName\":\"[hostpoolspecifics].HostPoolType\"},{\"originalName\":\"[HostUsers].Computername\"},{\"originalName\":\"[HostUsers].SessionHostName\"},{\"originalName\":\"[HostUsers].HostPool\"},{\"originalName\":\"[HostPools].LastUpdateStatus\"},{\"originalName\":\"[HostPools].LastUpdateTime\"},{\"originalName\":\"[hostpoolspecifics]. LoadBalanceType\"}]}",
```

Comment:
Let's output the `AssignedUser` column matching the casing of other columns: `Assigned user`

Reply:
Updated the column name to 'Assigned user'.